docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,668 +1,90 @@
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Edit agent tokens</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Revoke agent tokens</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td colspan="5">
-        <strong>USER TOKENS</strong>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        View and create own{" "}
-        <a href="/dagster-cloud/account/managing-user-agent-tokens#managing-user-tokens">
-          user tokens
-        </a>
-      </td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>List all user tokens</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Revoke all user tokens</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td colspan="5">
-        <strong>USER MANAGEMENT</strong>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <a href="/dagster-cloud/account/managing-users">View users</a>
-      </td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Add users</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Edit users</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td>Remove users</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-red-50">❌</td>
-      <td className="bg-green-50">✅</td>
-    </tr>
-    <tr>
-      <td colspan="5">
-        <strong>WORKSPACE ADMINISTRATION</strong>
```

Comment:
not new in this PR but 'workspace' is probably not right here - 'deployment administration'? And the specific "Edit workspace" should probably be up with Code Locations now